pkg/internal/export/otel/traces.go

```diff
@@ -567,6 +568,10 @@ func traceAttributes(span *request.Span, optionalAttrs map[attr.Name]struct{}) [
 			request.ServerPort(span.HostPort),
 		}
 	case request.EventTypeSQLClient:
+		attrs = []attribute.KeyValue{
```

Comment:
I guess I have to add this for Kafka, right?

Reply:
Yes that's right, we need to specify which attributes will be ON by default for kafka. We tend to select by default the attributes which will not cause cardinality explosion. Kafka operation is definitely OK to add in metrics, I think topic is too, although there will be customers with many topics. For those customers we can recommend that they remove the attribute by changing the config.